app/controllers/subjects_controller.rb

```diff
@@ -0,0 +1,74 @@
+class SubjectsController < ApplicationController
+  before_action :set_subject, only: [:show, :edit, :update, :destroy]
+
+  # GET /subjects
+  # GET /subjects.json
+  def index
+    @subjects = Subject.all
+  end
+
+  # GET /subjects/1
+  # GET /subjects/1.json
+  def show
+  end
+
+  # GET /subjects/new
+  def new
+    @subject = Subject.new
+  end
+
+  # GET /subjects/1/edit
+  def edit
+  end
+
+  # POST /subjects
+  # POST /subjects.json
+  def create
+    @subject = Subject.new(subject_params)
+
+    respond_to do |format|
+      if @subject.save
+        format.html { redirect_to @subject, notice: 'Subject was successfully created.' }
```

Comment:
Did RuboCop flag any of these lines for being too long? It flagged them on my. [Ruby styleguide](https://github.com/bbatsov/ruby-style-guide#80-character-limits) is to keep char length to 80. You can checkout my merged PR to see how I formatted the longer lines in accordance to the guide.